packages/react-provider/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@rarimo/react-provider",
+  "version": "1.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/rarimo/js-sdk/tree/main/packages/react-provider"
+  },
+  "license": "MIT",
+  "types": "./dist/index.d.ts",
+  "main": "./dist/react-provider.cjs.js",
+  "module": "./dist/react-provider.es.js",
+  "exports": {
+    ".": {
+      "import": "./dist/react-provider.es.js",
+      "require": "./dist/react-provider.cjs.js"
+    }
+  },
+  "scripts": {
+    "vite-dev-server": "vite",
+    "vite-build": "vite build",
+    "start": "cross-env VITE_APP_ENVIRONMENT=development yarn vite-dev-server",
+    "build": "yarn clean && cross-env VITE_APP_ENVIRONMENT=production yarn vite-build",
+    "clean": "rm -rf dist"
+  },
+  "dependencies": {
+    "@rarimo/provider": "workspace:^",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

Comment:
remove from deps, and add to devDeps and peerDeps, probably after it u can remove it from rollupOptions from vite config